submissions/m-ruslan/memory-pair-game/app.js

```diff
@@ -87,19 +92,18 @@ const flip = (cardID) => {
   document.querySelector("#" + cardID).classList.toggle("flip");
 };
 
-const match = (card1ID, card2ID) => {
-  document.querySelector("#" + card1ID).classList.add("hidden");
-  document.querySelector("#" + card2ID).classList.add("hidden");
-  qOfCardsLeft -= 2;
+const hideMatchedCards = (firstCardID, secondCardID) => {
+  document.querySelector("#" + firstCardID).classList.add("hidden");
```

Comment:
use template strings in such cases
```suggestion
  document.querySelector(`#${firstCardID}`).classList.add("hidden");
```

Reply:
@babayK0 thank you) could you please check my changes one more time?